modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -227,6 +227,39 @@ uses:
 
 REST Connect generates metadata for each operation based on your schema definition in the request and response for each method in your RAML. REST Connect cannot generate metadata based on examples in the RAML.
 
+Since only one input/output type is supported for each operation, REST Connect selects the first one that was declared. You can change this behavior by using the 'default' property from the REST Connect library. 
```

Comment:
Per doc standard, please replace Since with Because. The rest of the content looks good.

Reply:
fixed